docs/docs/components/textinput.md

```diff
@@ -117,6 +114,11 @@ secureTextEntry: boolean = false;
 // Should spell checking be applied to contents?
 spellCheck: boolean = [value of autoCorrect];
 
+// iOS and Windows only property for controlling when the clear button should appear on the right side of the text view.
+// Default behavior is dependent on platform: equivalent to 'never' on iOS, and 'always' on Windows.
```

Comment:
Would it be so bad to have the default be consistent? We could explicitly set 'never' if the incoming prop is undefined in nativecommon/TextInput

Reply:
We could, it's just that I'm torn. The default UWP behavior is something Windows users got accustomed to.

Reply:
I don't feel strongly about it... it's just an implementation detail that a dev needs to know and think about.

Reply:
I'm nearly done with the react-native-windows implementation of it. Default mode (if no value is provided) will be the behavior as it is right now, so nothing to worry about for devs who aren't aware of this prop. I've also ensured the 4 modes behave exactly like their iOS counterparts.